spec/features/indexing_custom_document_spec.rb

```diff
@@ -74,7 +74,6 @@
 
     it '#level' do
       expect(doc['level_ssm'].first).to eq 'collection'
-      expect(doc['level_sim'].first).to eq 'Collection'
```

Comment:
No problem, I though that a duplicate line had been created in error, I see now that they were different. I'll fix. Thanks!